backend/controllers/fileController.js

```diff
@@ -0,0 +1,100 @@
+const {storage, moveFile} = require('../util/uploadUtil')
+const {ProjectFile} = require('../model/db')
+var fs = require('fs'), async = require('async');
+const path = require('path')
+var multer  = require('multer')
+
+
+upload = multer({storage : storage});
+
+addFile = async function(req, res){
+	const obj = JSON.parse(JSON.stringify(req.body));
+	projectId = obj.projectId
+	file = req.files
+	try{
+		for(var i = 0; i < file.length; i++){
+		currentFile = file[i]
+		oldPath = `./uploads/${currentFile.filename}`
```

Comment:
what is the use of oldpath and new path? also it would be better if you fixed the indentation

Reply:
it's for moving project-file from uploads directory to its spesific directory